### PR TITLE
Adding sbg_driver to documentation index for distro

### PR DESCRIPTION
**Pull request by ThomasLeMezo**

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -4927,6 +4927,21 @@ repositories:
...
     status: developed
+  sbg_driver:
+    doc:
+      type: git
+      url: https://github.com/ENSTABretagneRobotics/sbg_ros_driver.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/ENSTABretagneRobotics/sbg_ros_driver-release.git
```
Comment (dirk-thomas) [inline kinetic/distribution.yaml]: This repo doesn't seem to exist? Have you followed the bloom tutorials to create a release?

Comment (ThomasLeMezo) [inline kinetic/distribution.yaml]: I'm currently working on it, should be done very soon (probably less than an hour)

**Comment by dirk-thomas** [comment]
This needs to be rebased against master (with the release added in a separate PR) to add the doc and source entries.

**Comment by ThomasLeMezo** [comment]
Ok, thanks for the speed of the support. I will make a new PR with the right info

**Pull request closed without merging**